translate/tools/pocount.py

```diff
@@ -55,14 +55,14 @@ def calcstats_old(filename):
     except ValueError as e:
         logger.warning(e)
         return {}
-    units = filter(lambda unit: unit.istranslatable(), store.units)
+    units = list(filter(lambda unit: unit.istranslatable(), store.units))
```

Comment:
Actually I hate more `lambda` than `filter`, but that is just some personal preference of mine.